app/views/events/schedule.html.erb

```diff
@@ -21,7 +21,7 @@
         <tr class="gameday">
           <th colspan="6">
             <b>
-              <%= t '.gameday' %> <%= match.gameday %>
+              <%= t '.gameday' %> <%= match.gameday %> <%= "#{ (@event.date_for_gameday match.gameday).strftime("%d.%m.") } #{t('.to')} #{((@event.date_for_gameday match.gameday+1)-1.day).strftime("%d.%m.")}"%>
```

Comment:
I think having the gameday and dates in one Tag is fine for now. We might have to seperate them later for more flexibility when styling them.

Reply:
I didn't want to waste time styling when we have a whole team assigned to that :P